ideascube/monitoring/forms.py

```diff
@@ -110,24 +113,28 @@ class LoanForm(forms.ModelForm):
 
     def clean_specimen(self):
         barcode = self.cleaned_data['specimen']
-        if Loan.objects.due().filter(specimen__barcode=barcode).exists():
+        if (Loan.objects.due().filter(specimen__barcode=barcode).exists()
+         or Loan.objects.due().filter(specimen__serial=barcode).exists()):
```

Comment:
I know you disagree with pep8 on this specific matter of alignment of conditions.

For now I think the pep8 way is better than yours, but I'm curious: how would you have written the above two lines with a `and` instead of the `or`?